pkg/template/template.go

```diff
@@ -142,10 +142,13 @@ func (r ReleaseSuite) ComponentReleaseVersion(repo string) string {
 
 func markdownHyperlinksToHTMLHyperlinks(sectionItem string) string {
 	linkTemplate := `<a href="%s">%s</a>`
+	linkTemplateNewWindow := `<a href="%s" target="_blank">%s</a>`
 
 	markdownRegex := regexp.MustCompile(`\[(.*?)\]\((.*?)\)`)
 	nameRegex := regexp.MustCompile(`\[(.*)\]`)
 	urlRegex := regexp.MustCompile(`\((.*)\)`)
+	conjurDocsURL := regexp.MustCompile(`docs\.conjur\.org`)
```

Comment:
Thoughts? on collapsing `conjurDocsURL` and `cyberarkDocsURL` into
```
docsURLRegex := regexp.MustCompile(`docs\.(conjur|cyberark)\.org`)
```

Then the condition can be:
```
if docsURLRegex.MatchString(url) {
    ...
}
```

Reply:
nice suggestion! I'll take it